examples/progressive-web-app/.npmrc

```diff
@@ -0,0 +1 @@
+auto-install-peers=true
```

Comment:
Is this necessary? Ideally we don't need to include an `.npmrc` for examples.

Reply:
PNPM failed on peers. It won't break anything but this will install peers automatically and prevent the error. Another option would be `strict-peer-dependencies=false` which does not install peers but also doesn't fail.

But in the end I can see why you wouldn't want an `.npmrc` and it's not related to the fix. Feel free to undo it.